spanner/ddl/ast.go

```diff
@@ -197,6 +227,20 @@ func (ci CreateIndex) PrintCreateIndex(c Config) string {
 	return fmt.Sprintf("CREATE INDEX %s ON %s (%s)", c.quote(ci.Name), c.quote(ci.Table), strings.Join(keys, ", "))
 }
 
+// PrintForeignKeyAlterTable unparses the foreign keys using ALTER TABLE.
+func (k Foreignkey) PrintForeignKeyAlterTable(c Config, tableName string) string {
```

Comment:
Should there be a test for this?
I think we test all public functions in ast.go.

Reply:
yes, i have added tests for this